KDUMP test port

mbivolan opened the pull request.

chvalean commented on Testscripts/Linux/KDUMP-Config.sh:
```diff
@@ -0,0 +1,401 @@
+            fi
+        ;;
+        *)
+            msg=
```
to be removed

adriansuhov commented on Testscripts/Linux/KDUMP-Config.sh:
```diff
@@ -0,0 +1,401 @@
+
+    kdump_commandline_arguments=$(grep KDUMP_COMMANDLINE_APPEND $kdump_sysconfig |  sed 's/KDUMP_COMMANDLINE_APPEND="//g' | sed 's/"//g')
+
+
```
2 blank lines

adriansuhov commented on Testscripts/Linux/KDUMP-Config.sh:
```diff
@@ -0,0 +1,401 @@
+#
+#######################################################################
+RhelExtraSettings(){
+
```
Try to remove blank lines where is possible or doesn't break code readability

asofron commented on Testscripts/Linux/KDUMP-Config.sh:
```diff
@@ -0,0 +1,401 @@
+        ;;
+        *)
+            msg=
+            LogErr "Warning: Distro '${distro}' not supported. Kexec-tools failed to install."
```
Replace with LogWarn

adriansuhov replied:
LogWarn is not available for bash scripts, only for ps1 afaik

asofron commented on Testscripts/Linux/KDUMP-Config.sh:
```diff
@@ -0,0 +1,401 @@
+    else
+        echo path $dump_path >> $kdump_conf
+        LogMsg "Success: Updated the path to /var/crash."
+        UpdateSummary "Success: Updated the path to /var/crash."
```
UpdateSummary should be removed

Commit:
HyperV,Azure: Added KDUMP tests

rmsivakanth committed:
Added new test to verify SRIOV failsafe for DPDK (#214)

* Added new test to verify SRIOV failsafe for DPDK

* Addressed all comments on PR#214

* Minor fix for indentation and pps results

ader1990 committed:
hyperv: Added VMBus tests (#218)

ader1990 committed:
hyperv: Add LIS NMI tests (#217)

2 new test casese added:
NMI_Verify_Interrupt (Checks if the NMI is properly sent to the linux vm)
NMI_Verify_Failed_Interrupt (Checks that the NMI cannot be sent to the vm when the vm is in stopped, paused or suspended states)

iamshital committed:
Added new test : Netperf for checking PPS performance (#200)

* Added new test : Netperf for checking PPS performance

This commit adds following variants of the test :
1. PERF-NETWORK-TCP-NETPERF-SINGLE-PPS-Synthetic
2. PERF-NETWORK-TCP-NETPERF-SINGLE-PPS-SRIOV
3. PERF-NETWORK-TCP-NETPERF-MAX-PPS-Synthetic
4. PERF-NETWORK-TCP-NETPERF-MAX-PPS-SRIOV

* Addressing comments on PR #200
